views/auth/local/login.ejs

```diff
@@ -1,25 +1,55 @@
-<div class="grid-container">
-  <div class="grid-row">
+<div class="grid-container login-block">
+
+  <div class="grid-row ">
     <div class="tablet:grid-col-6 tablet:grid-offset-3 desktop:grid-col-4 desktop:grid-offset-4">
-      <h1 class="text-center">Log in</h1>
-      <% if (locals.unauthorized) { %>
-      <div class="alert alert-danger">
-        Invalid email and/or password.
+      <div class='text-center'>
```

Comment:
I'm not sure why our pretty printer didn't pick it up, but I'm seeing an inconsistent use of single vs double quotes in this file, I'll change them all to double quotes for consistency with the rest of the codebase.